packages/pestudio.vm/pestudio.vm.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>pestudio.vm</id>
-    <version>9.61.20250219</version>
+    <version>0.0.0.20250219</version>
```

Comment:
@PrajeetGuha can you remove this change and create a separate PR? We can merge this PR without the CI to test that mftecmd is updated. But this means that the packages won't be pushed to MyGet. It is ok for mftecmd as the automatic update will push it. But not for pestudio.

Sorry, I think I said before we could leave the change.